MRN-Code/coinstac
# Home Screen Runs

# Problem
- Closes #363 
- Closes #411 
- Run items could use additional polishing
- Warning messages on missing consortia list items don't provide enough guidance.
# Solution
- Create common component - `run-list.jsx` for use across three different places runs are displayed, including Home screen
- Added pipeline name to run list item
- Added buttons to map data if local data mapping missing to both consortia list items and run items
- Added button to set active pipeline on consortia items if no activePipelineId set
- Fixed bug resulting in pipeline creation being called twice
- Added listener to pipeline state emitter in both UI and pipeline server
- Added GraphQL route to update run state
- Display side-by-side remote and local pipeline state 
# Testing
- Run test-setup in `coinstac-api-server`
- Start application and notice single Run listed on home page.
- Start another run from the consortia list page
- Switch back to home screen to see the new run *In Progress* with state updates.
